components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -669,6 +683,75 @@ private void addOrUpdateOIDCSpProperty(OAuthAppDO oauthAppDO,
                 TOKEN_BINDING_VALIDATION, String.valueOf(oauthAppDO.isTokenBindingValidationEnabled()),
                 prepStatementForPropertyAdd, preparedStatementForPropertyUpdate);
 
+        addOrUpdateOIDCSpProperty(preprocessedClientId, spTenantId, spOIDCProperties,
+                TOKEN_AUTH_METHOD, String.valueOf(oauthAppDO.getTokenEndpointAuthMethod()),
```

Comment:
As these getters used are returning String values, cant we straight away get the value from the getters without passing it to String.valueOf() method. Please check the other places as well in the PR.

Reply:
fixed with 09b42394545a8f7aad2e358fe50edf8e28e75047